assets/js/components/activation/activation-main.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * Activation Main component.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import Button from '../button';
+import Logo from '../logo';
+import Optin from '../optin';
+
+export function ActivationMain( { setupURL, onStartSetup, startSetupButtonText } ) {
+	return (
+		<div className="mdc-layout-grid">
+			<div className="mdc-layout-grid__inner">
+				<div className="
+						mdc-layout-grid__cell
+						mdc-layout-grid__cell--span-12
+					">
+					<Logo />
+
+					<h3 className="googlesitekit-heading-3 googlesitekit-activation__title">
+						{ __( 'Congratulations, the Site Kit plugin is now activated.', 'google-site-kit' ) }
+					</h3>
+
+					<div className="googlesitekit-opt-in googlesitekit-activation__opt-in">
+						<Optin />
+					</div>
+
+					<Button
+						id="start-setup-link"
+						className="googlesitekit-start-setup googlesitekit-activation__button"
+						href={ setupURL }
+						onClick={ onStartSetup }
+					>
+						{ startSetupButtonText }
+					</Button>
+				</div>
+			</div>
+		</div>
+	);
+}
+
+ActivationMain.propTypes = {
+	setupURL: PropTypes.string.isRequired,
+	onStartSetup: PropTypes.func,
+	startSetupButtonText: PropTypes.string.isRequired,
```

Comment:
I feel like we should name these props more action-agnostic now, e.g. `primaryButtonURL`, `onPrimaryButtonClick`, `primaryButtonLabel`.

Reply:
My initial thought is that using abstract prop names might make it a bit more confusing from the consuming component side (at least how it is now). This implies that there could be props for a secondary action or that the primary action may not be starting setup, but something else. In this case, I think it makes more sense to have explicit props, and the `ActivationMain` component can decide what is primary or secondary rather than the parent component needing to have knowledge about the inner workings of it. I could see this changing with the incoming changes once we have a better idea how the component will be used. What do you think?

Reply:
Hmm, we could also name it just `button...` instead of `primaryButton...`, if that is the point of confusion. I think by leaving out the `primary` it should be clear. There is no secondary action here anyway :)

Reply:
Sure that works. Updated!